specify/specify7
# Display synonyms node when hovering

Fixes #407


### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- go to any tree 
- hover over a node that has synonyms
- [ ] verify that the synonyms are being displayed
